### PR TITLE
editoast: front: uses editoast endpoints instead of api endpoints for study and scenarios

### LINKED ISSUE #3698: editoast: front: replace study and scenario api endpoints by editoast endpoints
replace all the call to api endpoints that are linked to project/studies/scenarios by a call to editoast endpoints

### PR DESCRIPTION
**Pull request by younesschrifi**
Close #3698 

As we modify a lot of endpoints (all the endpoints related to project, scenario and study), we do not migrate them to RTK to avoid breaking the app. The migration will be done in following PRs.

### CODE REVIEW

**Review by bloussou — COMMENTED** [review]
Can you change the integration test `test_scenario.py` ? And adapt fixtures in `conftest.py` to use editoast endpoints ?

**Review by flomonster — DISMISSED** [review]
LGTM. Thanks for your PR!

**Comment by younesschrifi** [comment]
> conftest

Done

**Review by eckter — DISMISSED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -120,22 +156,35 @@ def west_to_south_east_path(small_infra: Infra, fast_rolling_stock: int) -> Trai
...
+                            "track_section": "TH1",
+                            "geo_coordinate": [-0.095104854807785, 49.484],
+                        }
+                    ],
                 },
             ],
             "rolling_stocks": [fast_rolling_stock],
         },
     )
+    print(response.json())
```
Comment (bloussou) [inline tests/conftest.py]:
```suggestion
```

File: tests/tests/test_scenario.py [hunk 2]
```diff
@@ -26,9 +27,10 @@ class _ScenarioResponse:
 
 def test_get_scenario(small_scenario: Scenario):
     response = requests.get(
-        API_URL
+        EDITOAST_URL
         + f"projects/{small_scenario.project}/studies/{small_scenario.op_study}/scenarios/{small_scenario.scenario}/"
     )
+    print(response.content)
```
Comment (bloussou) [inline tests/tests/test_scenario.py]:
```suggestion
```

**Review by bloussou — DISMISSED** [review]
Just some print to remove in tests otherwise test part looks ok :heavy_check_mark:

**Review by anisometropie — DISMISSED** [review]

**Review by anisometropie — APPROVED** [review]

**Review by bloussou — APPROVED** [review]
Looks good :tada: Is there a plan to move to rtk endpoint after this PR ?

**Review by flomonster — APPROVED** [review]
Thanks for this PR!

**Review by eckter — APPROVED** [review]

**Pull request merged**